libs/components/packages/src/schematics/migrations/update-8/update-polyfill/schema.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "UpdatePolyfill",
+  "title": "Update polyfill to switch to config",
+  "type": "object"
```

Comment:
Do we need the schema for an update schematic?

Reply:
I've added it to the collection so it's easier to run apart from an update, specifically with the monorepo in mind. It needs a schema to be run separately.